tools/Dockerfiles/pki_build

```diff
@@ -0,0 +1,33 @@
+FROM fedora:latest
+
+# Install generic dependencies to build jss and pki
+RUN true \
+        && dnf update -y --refresh \
+        && dnf install -y dnf-plugins-core gcc make rpm-build \
+        && dnf copr -y enable ${JSS_4_6_REPO:-@pki/master} \
+        && dnf build-dep -y jss pki-core \
```

Comment:
builddep is covered again in line 21 and 22, which will install any missing/updated deps. Do we want this line?

Reply:
This pulls in 99% of the deps in the first layer. The layer is cleaned (see `dnf clean -y all` -- though I had to keep `tzinfo` and the other directories unlike the other layers because PKI build fails without it for some reason).

This results in a smaller first layer with all the deps, and then only what is new (in these two versions of JSS/PKI) is pulled in during a separate layer.